src/api/svg.ts

```diff
@@ -596,6 +732,11 @@ const parseAttributes = (
     const inner = newInherited.fontFamily.match(/^"(.*?)"|^'(.*?)'/);
     if (inner) newInherited.fontFamily = inner[1] || inner[2];
   }
+  
+  if (newInherited.strokeWidth) {
+    newInherited.strokeWidth = newConverter.size(newInherited.strokeWidth, newInherited.strokeWidth).width
```

Comment:
Maybe you should use the max between (xx.width, xx.height, 1), xx being the result of newConverter.size()